cmd/user-list/main.go

```diff
@@ -31,64 +33,52 @@ func main() {
 		log.Fatal("Could not load config options.", err)
 	}
 
-	db.MustOpen()
-	defer db.MustClose()
-
-	if args.Course == "" {
-		listServerUsers(args.Emails, args.Table)
-	} else {
-		listCourseUsers(args.Emails, args.Course, args.Table)
+	err = listServerUsers(args.Emails, args.Table)
+	if err != nil {
+		log.Fatal("Failed to list server users.", err)
 	}
 }
 
-func listServerUsers(emails []string, table bool) {
-	users, err := db.GetServerUsers()
+func listServerUsers(emails []string, table bool) error {
+	request := users.ListRequest{}
+
+	response, err := cmd.SendCMDRequest(`users/list`, request)
 	if err != nil {
-		log.Fatal("Failed to get server users.", err)
+		return fmt.Errorf("Failed to send the list server users CMD request: '%w'.", err)
 	}
 
-	if len(emails) > 0 {
-		newUsers := make(map[string]*model.ServerUser, len(emails))
-		for _, email := range emails {
-			newUsers[email] = users[email]
-		}
+	var responseContent users.ListResponse
+	util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
 
-		users = newUsers
+	if len(emails) > 0 {
+		response.Content = filterUsersByEmail(responseContent, emails)
+		util.MustJSONFromString(util.MustToJSON(response.Content), &responseContent)
 	}
 
 	if table {
 		fmt.Println(strings.Join(model.SERVER_USER_ROW_COLUMNS, "\t"))
-		for _, user := range users {
+		for _, user := range responseContent.Users {
 			fmt.Println(strings.Join(user.MustToRow(), "\t"))
 		}
 	} else {
-		fmt.Println(util.MustToJSONIndent(users))
+		cmd.PrintCMDResponse(response, users.ListResponse{}, args.Short)
 	}
-}
 
-func listCourseUsers(emails []string, courseID string, table bool) {
-	course := db.MustGetCourse(courseID)
+	return nil
+}
 
-	users, err := db.GetCourseUsers(course)
-	if err != nil {
-		log.Fatal("Failed to get course users.", err)
+func filterUsersByEmail(userList users.ListResponse, emails []string) users.ListResponse {
+	emailSet := make(map[string]struct{}, len(emails))
```

Comment:
This function will probably go away based on earlier comments,
but I just wanted to note that for sets you should probably use either `any` or `bool` for the value types in your map (and `nil` or `true` respectively as the placeholder values).